website/docs/d/scc_posture_latest_scans.html.markdown

```diff
@@ -0,0 +1,71 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_scc_posture_latest_scans"
+description: |-
+  Get information about list_latest_scans
+subcategory: "Security and Compliance Center"
+---
+
+# ibm_scc_posture_latest_scans
+
+Provides a read-only data source for list_latest_scans. You can then reference the fields of the data source in other resources within the same configuration using interpolation syntax.
+
+## Example Usage
+
+```hcl
+data "ibm_scc_posture_latest_scans" "list_latest_scans" {
+	scan_id = "262"
+}
+```
+
+## Argument Reference
+
+Review the argument reference that you can specify for your data source.
+
+* `scan_id` - (Optional, String) The ID of the scan.
+
+## Attribute Reference
+
+In addition to all argument references listed, you can access the following attribute references after your data source is created.
+
+* `id` - The unique identifier of the list_latest_scans.
+* `first` - (Optional, List) The URL of the first page of scans.
+Nested scheme for **first**:
+	* `href` - (Optional, String) The URL of the first page of scans.
+
+* `last` - (Optional, List) The URL of the last page of scans.
+Nested scheme for **last**:
+	* `href` - (Optional, String) The URL of the last page of scans.
+
+* `latest_scans` - (Optional, List) The details of a scan.
+Nested scheme for **latest_scans**:
+	* `scan_id` - (Optional, String) The ID of the scan.
+	* `scan_name` - (Optional, String) A system generated name that is the combination of 12 characters in the scope name and 12 characters of a profile name.
+	* `scope_id` - (Optional, String) The ID of the scan.
+	* `scope_name` - (Optional, String) The name of the scope.
+	* `profile_id` - (Optional, String) The ID of the profile.
+	* `profile_name` - (Optional, String) The name of the profile.
+	* `profile_type` - (Optional, String) The type of profile. To learn more about profile types, check out the [docs] (https://cloud.ibm.com/docs/security-compliance?topic=security-compliance-profiles).
+	  * Constraints: Allowable values are: standard, authored, custom, standard_cv, temmplategroup, standard_certificate
```

Comment:
Supported values are **standard**, **authored**, **custom**, **standard_cv**, **temmplategroup**, **standard_certificate**.

check the spelling and naming convention of **temmplategroup**